tests/behaviour/expectations/utils.ts

```diff
@@ -88,3 +88,7 @@ export function toCairoInt256(val: number | bigint): [string, string] {
 }
 
 export const toCairoInt8 = (val: number | bigint) => BigInt.asUintN(8, BigInt(val)).toString();
+
+export function cairoUint256toBigIntHex(val: { low: string; high: string }): string {
```

Comment:
Let's change the name to `cairoUint256toHex`. Bigint in the name suggests that the returned value is of BigInt type which is not true